### PR TITLE
Feature vp 16 alloperators

### PR DESCRIPTION
**Pull request by chris-reejoy**

### CODE REVIEW

File: calculator-app/src/Buttonnumbers.js [hunk 1]
```diff
@@ -1,71 +1,89 @@
...
+  ));
   return (
-    // call "App", "App-header" and "ul-numbers" class in the App.css
-    <div className="App">
-      <header className="App-header">
-        <ul className="ul-numbers">
-          {/*to display "clear","Del" and "numbers" buttons here*/}
-          <button
+    <div className="buttonNumberMainDiv">
+
```
Comment (Mutheeswari) [inline calculator-app/src/Buttonnumbers.js:66]: Remove extra line

File: calculator-app/src/Buttonnumbers.js [hunk 2]
```diff
@@ -1,71 +1,89 @@
...
             className="button-cldel"
             onClick={() => removeString(digit)}
           >
-           {buttonName.delete}
-          </button>
-        </ul>
-        <ul className="ul-numbers">{listItems} </ul>
-      </header>
+            {buttonName.delete}
+          </button> */}
```
Comment (Mutheeswari) [inline calculator-app/src/Buttonnumbers.js:81]: Remove unwanted tags

File: calculator-app/src/index.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
 import App from "./App"; //importing App.js to call the component
+// import Appjson from "./Appjson"
```
Comment (Mutheeswari) [inline calculator-app/src/index.js]: Remove unwanted import line

**Review by RanjiniEswari — APPROVED** [review]

File: calculator-app/src/Buttonnumbers.js (hunk 2 above)
Comment (chris-reejoy) [reply calculator-app/src/Buttonnumbers.js:81]: done

**Pull request merged**